secret/secrets_manager_client_test.go

```diff
@@ -40,6 +40,17 @@ func TestSecretsManagerCreateAndDeleteSecret(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	Cleanup := func(out *secretsmanager.CreateSecretOutput) {
+		if out != nil && out.Name != nil && out.ARN != nil {
```

Comment:
You shouldn't have to check that `out.Name` is non-nil since all you need to delete the secret is `out.ARN`.